client/src/hooks/datafiles/useFileListing.js

```diff
@@ -8,23 +8,33 @@ function useFileListing(section = 'FilesListing') {
   const { query_string: queryString, filter } = parse(useLocation().search);
 
   const { data, loading, error, params, loadingScroll, reachedEnd } =
-    useSelector(
-      (state) => ({
-        data: state?.files?.listing?.[section]?.map((file) => ({
-          ...file,
-          disabled:
+    useSelector((state) => {
```

Comment:
This is a super slick rewrite, and as far as I can tell it works locally. But I think it would be cleaner and a little bit more "reactive" if you used the `useRename.js` file to set the state and the existing `operationStatus.move` that's used there.

Reply:
@jalowe13 Out of curiosity, was does the `RUNNING` status do?

Reply:
Rewrote to have useRename set the status of the task, and fixed operation status being named as move and not rename. Passed the status back into teh rename module.